legend-engine-executionPlan-execution-store-relational-connection/pom.xml

```diff
@@ -237,6 +241,10 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Why we need this?

Reply:
We don't need it anymore. I have removed it in the updated commit